plugins/logging/logging.cpp

```diff
@@ -31,9 +31,9 @@ static const char * compatibleVersions[] = {
 static const char * EclDefinition =
 "export Logging := SERVICE : time\n"
 "  dbglog(const string src) : c,action,entrypoint='logDbgLog'; \n"
-"  addWorkunitInformation(const varstring txt, unsigned code=0, unsigned severity=0, const varstring source='user') : ctxmethod,action,entrypoint='addWuException'; \n"
-"  addWorkunitWarning(const varstring txt, unsigned code=0, unsigned severity=1, const varstring source='user') : ctxmethod,action,entrypoint='addWuException'; \n"
-"  addWorkunitError(const varstring txt, unsigned code=0, unsigned severity=2, const varstring source='user') : ctxmethod,action,entrypoint='addWuException'; \n"
+"  addWorkunitInformation(const varstring txt, unsigned code=0, unsigned severity=0, unsigned audience=2, const varstring source='user') : ctxmethod,action,entrypoint='addWuException'; \n"
```

Comment:
What is value 2 ?  Can we use a MACRO name instead ?

Reply:
How do I use macros in function definitions used inside ECL? (I originally tried to use the MessageAudience from jscm.hpp but that didn't work).

Reply:
Is the 2 here meant to be MSGAUD_user ?  I think we leave it as is, just unfortunate we couldn't maybe define MACRO values for these, and use MACRO names in enum so they are available as MACRO or enum.

Reply:
2 is MSGAUD_user.